core/embed/rust/src/ui/model_tr/component/passphrase.rs

```diff
@@ -147,13 +147,7 @@ impl ChoiceFactoryPassphrase {
 }
 
 impl ChoiceFactory for ChoiceFactoryPassphrase {
-    fn get(&self, choice_index: u8) -> ChoiceItem {
```

Comment:
Do some of these moves of `get` within the same `impl` have some idea behind it? In some components it is done and not in others (and the order of `count` and `get` seems also random)

Reply:
no these are remainders of some experimentation, better to be reverted. the order should at least be same in all the components